internal/infrastructure/router/router.go

```diff
@@ -80,9 +80,9 @@ func configCommonMiddlewares(e *echo.Echo, c RouterConfig) {
 		LogStatus: true,
 		LogURI:    true,
 
-		// Forwards error to the global error handler, so it can decide
-		// appropriate status code.
-		HandleError: true,
+		// We need to set this true, to avoid double execution, first by the
+		// logger middleware, second by the echo framework internals.
+		HandleError: false,
```

Comment:
The comment says "set this true" but the code is setting it to false. 

But with the `if c.Response().Committed ` in the error handler,  the comment is now incorrect. 

I don't know if `HandleError` should be set to `true` or to `false` as I didn't reproduce a practical difference (all was commited).  But I'd incline to keeping to original code and comment - as there might be a situation where the response was not committed yet and it needs to be set e.g. to internal error.

Reply:
Double execution of error parsing? The error handle is committing the response by calling `JSON()` so the commit check prevents any double execution.

Reply:
Nevermind. Let's go with what you have.